services/api/db/serializers.py

```diff
@@ -0,0 +1,132 @@
+from rest_framework import serializers
+from db.models import User
+from django.utils.encoding import smart_str, force_bytes, DjangoUnicodeDecodeError
+from django.utils.http import urlsafe_base64_decode, urlsafe_base64_encode
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError
+
+#Method for validating password.
+def validate_confirm_password(password,confirm_password):
+    if password != confirm_password:
+        return False
+    return True
+
+class SignUpEndPointSerializer(serializers.ModelSerializer):
+    confirm_password = serializers.CharField(style={'input_type': 'password'}, write_only=True)
```

Comment:
```suggestion
    confirm_password = serializers.CharField(style={'input_type': 'password'}, write_only=True)
```
is the style ={'input_type': 'password'} required in the serialializer confirm_password field?

Reply:
for security purpose its required .